CoffeeChat/Controller/SchedulingPlacesViewController.swift

```diff
@@ -0,0 +1,289 @@
+            height: LayoutHelper.shared.getCustomVerticalPadding(size: 469)
+        )
+        let nextButtonSize = CGSize(
+            width: LayoutHelper.shared.getCustomVerticalPadding(size:175),
+            height: LayoutHelper.shared.getCustomVerticalPadding(size: 53)
+        )
+
+        titleLabel.snp.makeConstraints { make in
+            make.centerX.equalToSuperview()
+            make.top.equalTo(view.safeAreaLayoutGuide).inset(topPadding)
+        }
+
+        infoLabel.snp.makeConstraints { make in
+            make.centerX.equalToSuperview()
+            make.top.equalTo(titleLabel.snp.bottom).offset(infoPadding)
+        }
+
+        locationsCollectionView.snp.makeConstraints { make in
+            make.size.equalTo(collectionViewSize)
+            make.centerX.equalToSuperview()
+            make.top.equalTo(infoLabel.snp.bottom).offset(titlePadding)
+        }
+
+        nextButton.snp.makeConstraints { make in
+            make.size.equalTo(nextButtonSize)
+            make.centerX.equalToSuperview()
+            make.top.equalTo(locationsCollectionView.snp.bottom).offset(nextPadding)
+        }
+
+        backButton.snp.makeConstraints { make in
+            make.centerX.equalToSuperview()
+            make.top.equalTo(nextButton.snp.bottom).offset(backPadding)
+        }
+    }
+
+    // MARK: Button Action
+    private func updateNext() {
+        let enable = selectedCtownLocations.count + selectedCampusLocations.count > 2
+        nextButton.isEnabled = enable
+        if enable {
+            nextButton.backgroundColor = .backgroundOrange
+            nextButton.layer.shadowColor = UIColor.black.cgColor
+            nextButton.layer.shadowOffset = CGSize(width: 0.0, height: 2.0)
+            nextButton.layer.shadowOpacity = 0.15
+            nextButton.layer.shadowRadius = 2
+        } else {
+            nextButton.backgroundColor = .inactiveGreen
+            nextButton.layer.shadowColor = .none
+            nextButton.layer.shadowOffset = CGSize(width: 0.0, height: 0.0)
+            nextButton.layer.shadowOpacity = 0
+            nextButton.layer.shadowRadius = 0
+        }
+    }
+
+    @objc private func nextButtonPressed() {
+        // TODO implement
+    }
+
+    @objc private func backButtonPressed() {
+        // TODO implement
+    }
+
+
+}
+
+// MARK: - UICollectionViewDelegate
+extension SchedulingPlacesViewController: UICollectionViewDelegate {
+
+    func collectionView(_ collectionView: UICollectionView, didSelectItemAt indexPath: IndexPath) {
+        let section = locationSections[indexPath.section]
+        switch section {
+        case .campus(let locations):
+            selectedCampusLocations.append(locations[indexPath.row])
+        case .ctown(let locations):
+            selectedCtownLocations.append(locations[indexPath.row])
+        }
+        (collectionView.cellForItem(at: indexPath) as? SchedulingCollectionViewCell)?.changeSelection(selected: true)
```

Comment:
can we unwrap w/ `if-let` and then call `changeSelection`. this is a bit difficult to read